xonsh/parser.py

```diff
@@ -634,10 +633,10 @@ def p_varargslist(self, p):
         p1, p2 = p[1], p[2]
         p3 = p[3] if lenp > 3 else None
         p4 = p[4] if lenp > 4 else None
-        p5 = p[5] if lenp > 5 else None
+        #p5 = p[5] if lenp > 5 else None
         p6 = p[6] if lenp > 6 else None
         p7 = p[7] if lenp > 7 else None
-        p8 = p[8] if lenp > 8 else None
+        #p8 = p[8] if lenp > 8 else None
```

Comment:
If we want to delete these, should we just delete them (rather than commenting out)?